### PR TITLE
Update dependency requirment `pgtest=~1.3,>=1.3.1`

### LINKED ISSUE #507: outdated pgtest dependency
the pgtest dependency is fixed to 1.2.0 which has issues with newer ubuntu versions
https://github.com/aiidateam/aiida-quantumespresso/blob/develop/setup.json#L71

please upgrade to the latest one

### PR DESCRIPTION
**Pull request by sphuber**
Fixes #507 

Older versions have bugs on Ubuntu systems.

### CODE REVIEW

File: setup.json [hunk 1]
```diff
@@ -68,7 +68,7 @@
             "prospector==1.1.7",
             "pylint==1.9.4; python_version<'3.0'",
             "pylint==2.3.1; python_version>='3.0'",
-            "pgtest==1.2.0",
+            "pgtest~=1.3,>=1.3.1",
```
Comment (greschd) [inline setup.json:71]: Are you sure ``1.3.1`` fixes all the bugs, and ``1.3.2`` is not required?

Comment (sphuber) [reply setup.json:71]: This is what we use for `aiida-core` at least

Comment (greschd) [reply setup.json:71]: @ltalirz should know

Comment (sphuber) [reply setup.json:71]: Looking at the [commits](https://github.com/jamesnunn/pgtest/commits/master) after `1.3.1` they do not seem like critical fixes

Comment (greschd) [reply setup.json:71]: Ah, from looking at the diff 1.3.1 could potentially raise some `ResourceWarning`s, but that shouldn't break anything.

Comment (ltalirz) [reply setup.json:71]: yeah... 1.3.2 is safer but 1.3.1 will still work.

**Pull request merged**